lib/blocks.php

```diff
@@ -147,6 +150,49 @@ function gutenberg_reregister_core_block_types() {
 
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
 
+/**
+ * Registers block styles for a core block.
+ *
+ * @param string $block_name The block-name.
+ *
+ * @return void
+ */
+function gutenberg_register_core_block_styles( $block_name ) {
+	if ( ! current_theme_supports( 'split-block-styles' ) ) {
+		return;
+	}
+
+	$block_name = str_replace( 'core/', '', $block_name );
+
+	// Transform "freeform" to "classic due to folder-name inconsistency.
+	$block_name = 'freeform' === $block_name ? 'classic' : $block_name;
```

Comment:
Can't we just rename the folder?

Reply:
Sure we can... Should it be in a separate PR or do we want to just make that change here?

Reply:
Good point with the folder renaming 😃

Reply:
Done in https://github.com/WordPress/gutenberg/pull/25220/commits/ba8e3bf60636e270f6aac1eaf2914073bfadb685